www/docs/en/dev/guide/appdev/security/index.md

```diff
@@ -53,11 +53,21 @@ There are ways to approximate certificate pinning, such as checking the server's
 
 There are also plugins that can do true certificate pinning for some platforms, assuming your app is able to do all of its network requests using the plugin (i.e.: no traditional XHR/AJAX requests, etc).
 
+## Using TLS/SSL
+
+If your app communicates to an external server, it should be communicating using modern encryption standards. Use `https` protocol whenever is possible.
+
+[Let's Encrypt](https://letsencrypt.org/) is a free, automated, and open certificate authority provided by the nonprofit [Internet Security Research Group](https://www.abetterinternet.org/). Let's Encrypt will offer free standard certificates, which will be sufficient for most developers. Enterprise organizations may still want to use a traditional certificate authority that offers more advanced features such as [Extended Validation](https://en.wikipedia.org/wiki/Extended_Validation_Certificate) or [Organization Validation](https://en.wikipedia.org/wiki/Public_key_certificate#Organization_validation) certificates.
```

Comment:
I would not mention Extended Validation as it's essentially a thing of the past. Browsers stopped or will stop showing the special "green bar" and badges for EV and they no longer make sense.

Reply:
Is organization validation is still relevant?

Reply:
No idea TBH. I never came across those. I don't really understand what they are and why you would get those.

EV were just in my memoray for security news like https://www.troyhunt.com/extended-validation-certificates-are-really-really-dead/

Reply:
I think the purpose is that you have a certificate authority confirm the identity of an agent who is representing an organization. The certificate authority will put the organization name on the certificate. Microsoft for example uses one.

I've applied your suggestions about the EV and also corrected some unrelated grammer. :+1:

Reply:
Makes sense :+1